main.cpp

```diff
@@ -68,17 +19,36 @@ void testPairToNumber(
     TelCoColorCoder::MinorColor minor,
     int expectedPairNumber)
 {
+    using namespace TelCoColorCoder;
     int pairNumber = TelCoColorCoder::GetPairNumberFromColor(major, minor);
     std::cout << "Got pair number " << pairNumber << std::endl;
     assert(pairNumber == expectedPairNumber);
 }
+void  ToString()                          
```

Comment:
ToString needs to return the manual as a string, not just output to the console.

Reply:
updated